loadtest/user/userentity/actions.go

```diff
@@ -1329,3 +1329,15 @@ func (ue *UserEntity) RemoveCustomStatus(userID string) error {
 	}
 	return nil
 }
+
+func (ue *UserEntity) CreatePostReminder(userID, postID string, targetTime int64) error {
+	_, err := ue.client.SetPostReminder(&model.PostReminder{
+		TargetTime: targetTime,
+		PostId:     postID,
+		UserId:     userID,
+	})
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
This should have been caught by the linter, right?
```suggestion
	return err
```

Reply:
Not really. I was just being consistent with the rest of the codebase here.

Reply:
Ah, maybe we had this configured in Playbooks, then, I've always seen this flagged by the linter. Let's go with this pattern if it's used everywhere, then.

Reply:
I just remembered why I did this. Mostly this is to safe-guard against new code additions. If you want to add further logic down below, you would need to modify the old line as well. Not a big deal always, but can be missed sometimes.

Reply:
Yeah, I see the potential benefit when adding new code, although it's not that important from my point of view. Either way, I'm 0/5 on this, not a big deal.